Gems/Atom/Feature/Common/Code/Source/SplashScreen/SplashScreenFeatureProcessor.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <SplashScreen/SplashScreenFeatureProcessor.h>
+#include <Atom/RPI.Public/RPIUtils.h>
+#include <AzCore/Settings/SettingsRegistryImpl.h>
+
+namespace AZ::Render
+{
+    void SplashScreenFeatureProcessor::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
+        {
+            serializeContext
+                ->Class<SplashScreenFeatureProcessor, FeatureProcessor>()
+                ->Version(1);
+        }
+    }
+
+    void SplashScreenFeatureProcessor::AddRenderPasses(RPI::RenderPipeline* renderPipeline)
+    {
+        AZ::ApplicationTypeQuery appType;
+        AZ::ComponentApplicationBus::Broadcast(&AZ::ComponentApplicationBus::Events::QueryApplicationType, appType);
+        if (!appType.IsValid() || appType.IsEditor())
+        {
+            return;
+        }
+
+        bool enable = false;
+
+        auto settingsRegistry = AZ::SettingsRegistry::Get();
+        static const AZStd::string setregPath = "/O3DE/Atom/Feature/SplashScreenEnable";
```

Comment:
I'm tempted to recommend you put this in `SplashScreenFeatureProcessor.h` like
```
inline constexpr const char* SplashScreenEnableKey= "/O3DE/Atom/Feature/SplashScreenEnable";
```
but it is only used in 2 places so...